servant-lucid.cabal

```
@@ -20,7 +20,7 @@ copyright:           2015-2018 Servant Contributors
 category:            Web, Servant
 build-type:          Simple
 extra-source-files:  CHANGELOG.md
-tested-with: GHC ==8.6.5 || ==8.8.4 || ==8.10.7 || ==9.0.2 || ==9.2.8 || ==9.4.8 || ==9.6.6 || ==9.8.4
+tested-with: GHC ==8.6.5 || ==8.8.4 || ==8.10.7 || ==9.0.2 || ==9.2.8 || ==9.4.8 || ==9.6.6 || ==9.8.4 || ==9.10.2 || ==9.12.1 || ==9.12.2
```

No need for several minor versions, you can do:
```suggestion
tested-with: ==9.2.8 || ==9.4.8 || ==9.6.6 || ==9.8.4 || ==9.10.2 || ==9.12.2
```